regexes/device/mobiles.yml

```diff
@@ -2661,10 +2657,7 @@ Brondi:
     - regex: 'VICTORY(?:[ _-]?([2-6]))?(?:[);/ ]|$)'
       model: 'VICTORY $1'
 
-    - regex: 'Surfing Tab ([^;/]+) Build'
-      model: 'SURFING TAB $1'
-      device: 'tablet'
-    - regex: 'Surfing Tab ([^;/]+)(?:[);/ ]|$)'
+    - regex: 'Surfing Tab ([^);/]+)(?: Build|\))'
```

Comment:
Combining such regexes actually have the risk to match a model that includes `Build`.
If there would be a useragent including something like `; Surfing Tab 15 Build) AppleWebKit` the new regex would actually match `Surfing Tab 15 Build` instead of `Surfing Tab 15`. Guess that could be circumvented by performing the regex with ungreedy flag, but not sure if that's a good option

Reply:
this is the current version of ' master branch`, which gives this result  

```yml
user_agent: Mozilla/5.0 (Linux; Android 4.4.2; SURFING TAB 10.1 3G) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/80.0.3987.149 Safari/537.36
os:
  name: Android
  version: 4.4.2
  platform: ""
client:
  type: browser
  name: Chrome
  version: 80.0.3987.149
  engine: Blink
  engine_version: ""
device:
  type: tablet
  brand: Brondi
  model: SURFING TAB 10.1 3G) AppleWebKit
os_family: Android
browser_family: Chrome
```

this is from my logs

|id|value|created_at|
|--|----|----------|
|167845821|Mozilla/5.0 (Linux; Android 4.4.2; Surfing Tab C 3G) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/72.0.3626.105 Safari/537.36|2019-03-12 14:13:27|
|168054375|Mozilla/5.0 (Linux; Android 4.1.2; SURFING TAB A 3G Build/JZO54K) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/67.0.3396.87 Safari/537.36|2019-03-28 17:50:34|
|168928677|Mozilla/5.0 (Linux; Android 4.2.2; surfing tab 8 3G) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/71.0.3578.99 Safari/537.36|2019-06-21 22:43:15|
|169128155|Mozilla/5.0 (Linux; Android 4.4.2; SURFING TAB 10.1 3G Build/KOT49H) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/30.0.0.0 Safari/537.36|2019-07-25 20:21:37|
|169587989|Mozilla/5.0 (Linux; Android 4.4.2; Surfing Tab C 3G Build/KOT49H) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/30.0.0.0 Safari/537.36|2019-09-15 13:58:37|
|169926967|Mozilla/5.0 (Linux; U; Android 4.2.2; ru-ru; surfing tab 8 3G Build/SURFING-V1.0-2013.12.26) AppleWebKit/534.30 (KHTML, like Gecko) Version/4.0 Safari/534.30|2019-10-26 20:01:41|
|170445893|Mozilla/5.0 (Linux; Android 4.4.2; Surfing Tab C 3G) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/79.0.3945.93 Safari/537.36|2020-01-10 15:16:15|